doc/deployment/deploy-to-aks-with-acr-using-jenkins.md

```diff
@@ -0,0 +1,152 @@
+# Deploy to Azure Kubernetes Service using Jenkins
+
+This document shows how to deploy this todo app java project to Kubernetes cluster using Jenkins.
+
+On the Jenkins machine, it clones the toao-app-java-on-azure to local with **Git Plugin**, uses **Maven Plugin** to build out a `jar` file. 
+Using **Azure ACR PLugin**, Jenkins uploads the `jar` together with `Dockerfile` to Azure Container Registry. ACR Quick Build will build a docker image and host it when receiving the `Dockerfile` and `jar` file. 
```

Comment:
This, I believe is a copy and paste from the build from local workflow. Needs to be rewritten.